src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/InnerAssignmentCheckTest.java

```diff
@@ -49,7 +49,7 @@ public void testIt() throws Exception {
     }
 
     @Test
-    public void testMethod() throws Exception {
+    public void testInputInnerAssignmentMethod() throws Exception {
```

Comment:
The file name [InputInnerAssignmentMethod.java](https://github.com/checkstyle/checkstyle/blob/master/src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/innerassignment/InputInnerAssignmentMethod.java) doesn't seem accurate.
Please look for examples of ideal updates in the issue description

I would recommend something like : `InputInnerAssignmentNoViolationIterativeStatements`.java